core/src/main/scala/kafka/server/ReplicaFetcherManager.scala

```diff
@@ -32,6 +36,23 @@ class ReplicaFetcherManager(brokerConfig: KafkaConfig,
         clientId = "Replica",
         numFetchers = brokerConfig.numReplicaFetchers) {
 
+  newGauge(
+    "ReassignmentMaxLagOnFollower",
```

Comment:
Another way might be to use a tag to distinguish leader and follower. So the metric name is `ReassignmentMaxLag` in both cases.

Reply:
I agree that a tag might be better.